avoid compile error:
  CC      init/version.o                                                                                                                                                                      
  LD      init/built-in.o                                                                                                                                                                     
drivers/built-in.o: In function `rndis_flow_control':                                                                                                                                         
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1018: undefined reference to `is_rndis_ipa_supported'                 
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1018:(.text+0x45fd30): relocation truncated to fit: R_AARCH64_CALL26 a
gainst undefined symbol `is_rndis_ipa_supported'                                                                                                                                              
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1020: undefined reference to `u_bam_data_stop_rndis_ipa'              
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1020:(.text+0x45fd54): relocation truncated to fit: R_AARCH64_CALL26 a
gainst undefined symbol `u_bam_data_stop_rndis_ipa'                                                                                                                                           
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1027: undefined reference to `is_rndis_ipa_supported'                 
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1027:(.text+0x45fdc0): relocation truncated to fit: R_AARCH64_CALL26 a
gainst undefined symbol `is_rndis_ipa_supported'                                                                                                                                              
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1029: undefined reference to `u_bam_data_start_rndis_ipa'             
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:1029:(.text+0x45fde4): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `u_bam_data_start_rndis_ipa'
drivers/built-in.o: In function `rndis_msg_parser':
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:857: undefined reference to `is_rndis_ipa_supported'
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:857:(.text+0x4603f4): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `is_rndis_ipa_supported'
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:864: undefined reference to `u_bam_data_stop_rndis_ipa'
/home/pmos/build/src/android_kernel_samsung_msm8976-82d475a1952b9bbb6652b35e87671b5031bcf934/drivers/usb/gadget/rndis.c:864:(.text+0x460460): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `u_bam_data_stop_rndis_ipa'
make: *** [Makefile:819: vmlinux] Error 1


--- a/drivers/usb/gadget/f_qc_rndis.c
+++ b/drivers/usb/gadget/f_qc_rndis.c
@@ -29,6 +29,7 @@
 #include <linux/etherdevice.h>
 
 #include <linux/atomic.h>
+#include <linux/moduleparam.h>
 
 #include "u_ether.h"
 #include "u_qc_ether.h"
